### ISSUE TITLE
Problem with Installing 

### ISSUE THREAD

**Issue by nexondo** [bart/umbrel-status-screen#2]
Dear Bart,

thanks for the amazing work! <3

sadly i'm newbie and not able to install it correctly....sorry...

1. i managed to find the details file should i replace the whole content of it? and how is most easy way to replace everything with nano editor?

2. the last step is to change just the entries you mentioned in the service file?

thank you so much for helping me out!

best regards,
Jason

**Comment by bart** [comment]
Hey @nexondo sorry for my late reply. Had a couple of things to do the last few days.
1. You can simply replace the entire file by copying over umbrel-details via `cp` command
2. Exactly and afterwards restart Umbrel via Web Interface (or just reboot your Raspberry Pi)

Best of luck!

**Comment by nexondo** [comment]
Hi @bart thank you so much for your answer!

so i did the work
replaced the full umbrel-details

_by accident i created no baackup (.bak) could i get into trouble because of that?
like is an update causing issues or will the file replaced than anyways?_

**my service section now looks like this:**

```
Restart=always
ExecStart=/home/umbrel/umbrel/scripts/umbrel-os/umbrel-details
User=umbrel
Group=umbrel
StandardOutput=tty
TTYPath=/dev/tty1
RemainAfterExit=yes

[Install]
WantedBy=multi-user.target

```

**but my screen only shows:**
![IMG_5280](https://user-images.githubusercontent.com/119508310/210276588-0017cf44-0124-4887-9e50-56e73809c89b.jpeg)

**Comment by bart** [comment]
Hey @nexondo maybe the comment by [JarrydVanHoy](https://github.com/JarrydVanHoy) may help you?

https://github.com/bart/umbrel-status-screen/issues/3

**Comment by nexondo** [comment]
Hey @bart & @JarrydVanHoy,

thank you for all the work.
so after updating the umbrel-details and service section it still only shows me the starting services/dockers screen as above.
any ideas?

it looks like the service / details are not executed?
because i guess in the last step it's starting the service file which also refresehes the umbrel-detail code every 60 secs
but on my umbrel it stays at the image above....

thank you so much for your help! 🙏🎉

**Comment by JarrydVanHoy** [comment]
Hey @nexondo ,

What version of Umbrel OS are you running? My issues were related to some changes that Umbrel OS (or maybe app devs) had made as my problem was dealing with docker container names.

I am running version `0.5.3`.

Another thing to check is your `umbrel-connection-details.service` again.  Here is my [Service] section:
![image](https://user-images.githubusercontent.com/12432968/210856726-2011a35e-9cfc-480f-a429-9cb4256db112.png)

@bart linked my issue/thread of above, but other than container names I provided my own twist to bart's umbrel-details as I was having bleed over and wanted BTC price on there too.

Edit: I'll likely need to update my own version of umbrel-details again as my LND fees are about to bleed over. I'll come back and update my own issue if i do.

**Comment by nexondo** [comment]
Hi @JarrydVanHoy,

thank you su much for the instant reply :)

ok got it!
i'm running latest Version 0.5.3 as well.

i checked my docker container names are normal, so i changed it back to normal names: bitcoin & lightning
i also changed the service back to root username & group root

i know, that you made your own adjustments
i would like to as well, but for me there is displayed script at all :/

to be honest i have no idea why the script is not starting at all.

with my small knowledge about programming i think, that umbrel startup contains also the service file
which contains the command to execute the umbrel-details script and refresh it every 60secs which is modified with bart's script to ask for stats and display them.

so how can i figure out if the script is starting or not?
or how do i know if there is a issue while it tries to print the screen? because on the screen it always stays at the starting services but there should be at least something happening or not?

thank you really much!
best regards

**Comment by bart** [comment]
@nexondo Yeah there should follow a little more lines and finally a login prompt should appear. Maybe it's a permission issue of a file that should be executable. Maybe you can check your system logs for some warnings or errors?

**Comment by JarrydVanHoy** [comment]
@nexondo you could also try re-flashing UmbrelOS if you haven't gotten too far into you journey.
